content/docs/reactivesearch/react-searchbox/searchbase.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 'SearchBase API Reference'
+meta_title: 'API Reference for SearchBase'
+meta_description: 'SearchBase is a lightweight and platform agnostic search library for Elasticsearch.'
+keywords:
+    - apireference
+    - searchbase
+    - elasticsearch
+    - search library
+sidebar: 'docs'
+nestedSidebar: 'react-searchbox-reactivesearch'
+---
+
+## SearchBase
+
+It is a container component that wraps all the ReactiveSearch components together. It binds the backend app (data source) with the UI view components
```

Comment:
```suggestion
`SearchBase` is a container component that wraps all the Searchbox components together. It binds the backend index (data source) with the UI view components.
```

Reply:
@nikhil-varma When you copy from other docs, make sure to edit for context of the current doc.